### ISSUE TITLE
Error in test.lua

### ISSUE THREAD

**Issue by omkarparkhi** [facebook/fb-caffe-exts#6]
Hi,

I am trying to run `th torch2caffe/test.lua`. It seems to be failing with following error.

`I0303 14:42:44.412389   146 layer_factory.hpp:77] Creating layer caffe.LogSoftmax_2
F0303 14:42:44.412438   146 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: LogSoftmax (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Pooling, Power, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)`

Any suggestions as to whats going wrong?

**Comment by czhang96** [comment]
I'm also having this issue, does anyone have a solution?

**Comment by zlqhem** [comment]
I'm also having this issue.

**Comment by huyn** [comment]
because `caffe` does not support `LogSoftmax`, thus you need to replace it by an existed   
equivalent layer in caffe by yourself
